Dockerfile

```diff
@@ -0,0 +1,39 @@
+# syntax=docker/dockerfile:1.3
+
+ARG PHP_VERSION=7.4
+
+FROM php:${PHP_VERSION}-cli AS base
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV COMPOSER_FLAGS="--prefer-dist --no-interaction"
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+COPY docker/php/php.ini /usr/local/etc/php/php.ini
+
+RUN apt-get update && apt-get install -y \
+        git \
+        unzip \
+        zlib1g-dev \
+        libzip-dev \
+	--no-install-recommends && rm -r /var/lib/apt/lists/*
+
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin/ --filename=composer
+
+RUN pecl channel-update pecl.php.net \
+    && pecl config-set php_ini /usr/local/etc/php.ini \
+    && pecl install xdebug-beta \
+    && docker-php-ext-enable xdebug \
+	&& docker-php-ext-install zip
+
+FROM base AS input-mapping
+
+ENV LIB_NAME=input-mapping
```

Comment:
Zatim ti to prekazet nebude, ale ve chvili, kdy budes mit v tom monorepu 2 knihovny, ktere na sobe zavisi, budes potrebovat `ARG COMPOSER_MIRROR_PATH_REPOS=1`, aby composer nepouzival symlinky. Dal bych to sem radeji rovnou.